test/e2e/e2e.go

```diff
@@ -79,8 +84,12 @@ func RunE2ETests(t *testing.T) {
 	RunSpecsWithDefaultAndCustomReporters(t, "Metallb Operator Validation Suite", rr)
 }
 
-var _ = Describe("validation", func() {
+var _ = Describe("metallb", func() {
 	Context("Platform Check", func() {
```

Comment:
can you move this to the validation suite?

Reply:
Planned in a follow up PR. Would prefere to keep it separate to keep the pr's smaller and nicer to review